backend/src/database/tag/tag.service.ts

```diff
@@ -2,11 +2,16 @@ import { Injectable } from '@nestjs/common';
 import { PrismaService } from '../prisma.service';
 import { Tag } from '@prisma/client';
 
+const tagLimit: number = 5
+
 @Injectable()
 export class TagService {
-  constructor(private prisma: PrismaService) {}
+    constructor(private prisma: PrismaService) { }
 
     async createTag(tagname: string): Promise<Tag> {
+        if (tagname) {
+            tagname = tagname.toLowerCase()
+        }
         return this.prisma.tag.create({
```

Comment:
Dont we need to only add a tag to the database if it exists? 
Because in line 12, its checked if a tagname exists, and if so its turned lowercase.

Reply:
null?

Reply:
you're right, the check of null is unnecessary